pkg/sendconfig/common_workflows.go

```diff
@@ -57,8 +71,24 @@ func UpdateKongAdminSimple(ctx context.Context,
 		targetConfig, kongConfig.FilterTags, nil, lastConfigSHA, false,
 	)
 	if err != nil {
+		if diagnostic != (util.ConfigDumpDiagnostic{}) {
+			select {
+			case diagnostic.Configs <- util.ConfigDump{Failed: true, Config: *diagConfig}:
```

Comment:
```suggestion
			case diagnostic.Configs :<- util.ConfigDump{Failed: true, Config: *diagConfig}:
```

Reply:
Not sure what was behind the request here. We do indeed want to send to the channel here, not receive from it. Receiving happens in the diagnostic server loop. If you still think there should be a change here, please clarify and reopen.